webapp/components/search_bar.jsx

```diff
@@ -141,6 +141,8 @@ export default class SearchBar extends React.Component {
     handleSubmit(e) {
         e.preventDefault();
         this.performSearch(this.state.searchTerm.trim());
+        $(ReactDOM.findDOMNode(this.refs.search)).find('input').blur();
```

Comment:
no need to use `ReactDOM.findDOMNode` it can be `$(this.refs.search)`

+1 other than my comment